### ISSUE TITLE
DockBuilder methods not available

### ISSUE THREAD

**Issue by oliviadetrick** [apicici/cimgui-love#17]
As the title says, none of the DockBuilder* methods are available to use.  The referenced cimgui.h for the generator has them stripped out.  Can these be added back in?  I tried building the files myself with the official cimgui.h (with FILE stuff stripped out) and it's not working.  Were these removed for a reason?

**Comment by apicici** [comment]
I'm not familiar with the DockBuilder methods but it seems they are part of the internal API, which I'm not wrapping. I'm not planning on adding since I don't really use it, but I just tried to build cimgui with the internal API and it works (well, I haven't tried actually using any of the internal methods, but everything compiles and the LÖVE test file works).

What errors are you encountering? Is it related to the FILE stuff? I had to add this line at the beginning of `cdef.lua` for things to work:
```c
typedef struct FILE FILE;
```
